.gitlab-ci.yml

```
@@ -11,13 +11,15 @@ variables:
     rhel8/global
     openmpi/4.1.1/gcc-9.4.0-epagguv
     gcc/9.4.0/gcc-11.2.0-72sgv5z
+    hdf5/1.12.1
```

This particular module was built on RHEL7 so we should instead use
```suggestion
    hdf5/1.10.7/openmpi-4.1.1/gcc-9.4.0-krjrm53
```
We might need to switch to building the tests with `USE_MPI = TRUE` to make this work though.